plugins/modules/splunk_data_inputs_monitors.py

```diff
@@ -0,0 +1 @@
+./data_inputs_monitors.py
```

Comment:
there's no need of linking the file to `splunk_*`, you can directly name the module with `splunk_*` and there'd be no need for redirection. I am not sure why it was done earlier that way but I believe at this point it's unnecessary.